File: scratch/query.js
const sqlite3 = require('sqlite3').verbose();
const path = require('path');

const dbPath = path.join(__dirname, '..', 'data', 'radius.db');
const db = new sqlite3.Database(dbPath, (err) => {
    if (err) {
        console.error("Error connecting:", err);
        return;
    }
    console.log("Connected to", dbPath);
    
    db.all("SELECT * FROM nas", (err, rows) => {
        if (err) console.error("NAS error:", err);
        console.log("NAS Table:");
        console.table(rows);
        
        db.all("SELECT * FROM radcheck WHERE username='falisa'", (err, rows2) => {
            if (err) console.error("Radcheck error:", err);
            console.log("\nRadcheck Table for falisa:");
            console.table(rows2);
            
            db.all("SELECT * FROM radusergroup WHERE username='falisa'", (err, rows3) => {
                if (err) console.error("Radusergroup error:", err);
                console.log("\nRadusergroup Table for falisa:");
                console.table(rows3);
                
                db.close();
            });
        });
    });
});

File: scratch/query.py
import sqlite3
import sys
import os

try:
    db_path = os.path.join(os.path.dirname(__file__), '..', 'data', 'radius.db')
    conn = sqlite3.connect(db_path)
    c = conn.cursor()

    print("--- NAS Table ---")
    c.execute("SELECT * FROM nas")
    for row in c.fetchall():
        print(row)

    print("\n--- RADCHECK Table for falisa ---")
    c.execute("SELECT * FROM radcheck WHERE username='falisa'")
    for row in c.fetchall():
        print(row)

    conn.close()
except Exception as e:
    print(f"Error: {e}")
